uring/jvm/src/main/scala/fs2/io/uring/unsafe/UringCompletionQueue.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 Arman Bilge
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.netty.incubator.channel.uring
```

Comment:
Are we able to isolate the package trickery to just one file (or at least a small number of files)? I would ideally like to not leak that all over.